dashboards-observability/public/components/explorer/explorer.tsx

```diff
@@ -976,7 +976,9 @@ export const Explorer = ({
             name: selectedPanelNameRef.current,
             timestamp: currQuery![SELECTED_TIMESTAMP],
             applicationId: appId,
-            userConfigs: JSON.stringify(userVizConfigs[curVisId]),
+            userConfigs: userVizConfigs[curVisId]
```

Comment:
Are you checking if the property exists? If so then try using [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty)

Reply:
Okay!